### DISCUSSION TITLE
Muddle::link: obfuscate the title

### DISCUSSION THREAD

**Discussion by C2H6-383** [mokhosh/muddle#3] [Ideas]
Sometimes I need a link with the mail itself (`mailto:mail@example.com` as href) and the same mail as visible title/text for the link.

The current problem is that the title itself is not obfuscated and placing the mail in plain text would just void the purpose of this plugin.

In additon nesting a `muddle-text` element into a link element is not working properly (character escaping problems) and I also think this is not the best and cleanest strategy to eliminate the problem.

Do you think it is possible to just obfuscate the title alongside the mail adress? there should not be consequences for the end user as far I know, as all parts are deobfuscated in the browser.

**Comment by mokhosh** [comment]
That's a good idea. 
I will accept a pr with tests.

**Comment by mokhosh** [comment]
Done https://github.com/mokhosh/muddle/pull/14
